README.md

```diff
@@ -223,6 +223,18 @@ proxy[Comlink.releaseProxy]();
 
 If the browser supports the [WeakRef proposal], `[releaseProxy]()` will be called automatically when the proxy created by `wrap()` gets garbage collected.
 
+When using Comlink with a WebWorker, it is advised to only terminate the worker when the promise returned by `[releaseProxy]()` has resolved.
```

Comment:
Adding a note on _why_ this is advised is an important detail here.

My understanding is that if you terminate the worker before the proxy is released, then you leave in-flight promises in an unresolved state which can result in those promises holding into memory?